src/python/CMakeLists.txt

```diff
@@ -38,12 +38,33 @@ execute_process (
 file (TO_CMAKE_PATH "${PYTHON_PREFIX}" PYTHON_PREFIX)
 execute_process (
    COMMAND ${PYTHON_EXECUTABLE} -c
-       "import site, sys; sys.stdout.write(site.getusersitepackages().replace(site.getuserbase(), site.PREFIXES[-1]))"
+       "	\n
+import site, sys \n
+dirs = site.getsitepackages(); \n
+dir = dirs[0] \n
+for d in dirs: \n
+	if d.count(\"packages\"): \n
+		dir = d[d.index(\"packages\")-5:] \n
+		break \n
+py = \"${PYTHON_EXECUTABLE}\" \n
+py = py[py.rfind(\"/\")+1:] \n
+prefix = \"${PYTHON_PREFIX}\"[\"${PYTHON_PREFIX}\".rfind(\"/\"):] \n
+dir = \"{0}/lib/{1}/{2}\".format(prefix, py, dir) \n
+sys.stdout.write(dir)"
    OUTPUT_VARIABLE PYTHON_SITE_DIR
 )
-file (TO_CMAKE_PATH "${PYTHON_SITE_DIR}" PYTHON_SITE_DIR)
-string (REGEX REPLACE "^${PYTHON_PREFIX}/" ""
-   PYTHON_SITE_DIR "${PYTHON_SITE_DIR}")
+#"import site, sys; sys.stdout.write(site.getsitepackages()[0])"
+
+message(STATUS "python site dirs: ${PYTHON_SITE_DIR}")
+message(STATUS "python prefix: ${PYTHON_PREFIX}")
+
+if("${CMAKE_INSTALL_PREFIX}" STREQUAL "/usr")
```

Comment:
What if it's not `/usr`?

Could you please clarify the point of this whole block? The `PYTHON_SITE_DIR` variable is being set to `lib/python2.7/dist-packages` on my Debian 8 after this is executed, so the `install()` function will try to install it into current directory.